test/gurka/test_reachability.cc

```diff
@@ -0,0 +1,51 @@
+#include "gurka.h"
```

Comment:
nit: there is already a `test_reach` and a `test_route` gurka test, maybe this should just go in one of those?